ocs_ci/framework/pytest_customization/ocscilib.py

```diff
@@ -382,3 +382,12 @@ def pytest_runtest_makereport(item, call):
             call.start,
             call.stop
         )
+
+    # Get the performance metrics when tests fails for scale or performance tag
+    from tests.helpers import collect_performance_stats
```

Comment:
Cant you have imports on top of the code?

Reply:
Was hitting cyclic import error. So called here